pwndbg/commands/__init__.py

```diff
@@ -220,15 +224,21 @@ def fix(
     if isinstance(arg, pwndbg.dbg_mod.Value):
         return arg
 
+    frame = pwndbg.dbg.selected_frame()
+    target: pwndbg.dbg_mod.Frame | pwndbg.dbg_mod.Process = (
+        frame if frame else pwndbg.dbg.selected_inferior()
+    )
```

Comment:
Should we add a `pwndbg.dbg.evaluate_expression` method that does this and then calls `target.evaluate_expression()`?

Reply:
Given that this happens only twice in this PR, maybe it makes sense to wait and see how common that `pwndbg.dbg.evaluate_expression` would actually be? Pwndbg seems to use `parse_and_eval` very rarely outside of `gdblib`, and in most of those there might be better ways to do what's being done with it, especially with the LLDB port in mind.

Plus, that construction I feel communicates at very clearly that it's choosing to evaluate in a narrower, then broader context, in that order. So, if it's only really gonna be used these two times, I kind of prefer the longer form.

Either way, I feel like it's fairly easy to just add that in later if it that pattern ends up spreading out. But if you feel strongly about that, I can add it in, too.

Reply:
No need to add it now, I thought having a generic method in `dbg` that took care of this for you without having to think about the details of frames would be useful, but let's just add that when/if it becomes necessary.